cmd/noms/noms_sync.go

```diff
@@ -37,22 +37,22 @@ func setupSyncFlags() *flag.FlagSet {
 	syncFlagSet := flag.NewFlagSet("sync", flag.ExitOnError)
 	syncFlagSet.IntVar(&p, "p", 512, "parallelism")
 	spec.RegisterDatabaseFlags(syncFlagSet)
+	spec.RegisterVerboseFlags(syncFlagSet)
 	profile.RegisterProfileFlags(syncFlagSet)
 	return syncFlagSet
 }
 
 func runSync(args []string) int {
-	spec, err := spec.NewResolver()
-	d.CheckErrorNoUsage(err)
-	sourceStore, sourceObj, err := spec.GetPath(args[0])
+	resolver := spec.NewResolver()
+	sourceStore, sourceObj, err := resolver.GetPath(args[0])
 	d.CheckError(err)
```

Comment:
Often times, I see d.CheckError after a resolver failure, and then I see d.CheckErrorNoUsage() after others. Seems like we should do one or the other most of the time.